hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/OMBucketCreateResponse.java

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
@bharatviswa504 why is this change a part of the DoubleBuffer patch? This patch should just introduce the DoubleBuffer.

Reply:
Added them for testing purpose to test DoubleBuffer Implementation.
Actual usage and the classes will be changed further according to needs of HA in further Jira.

In this PR https://github.com/apache/hadoop/pull/827 handled bucket related operations.